ha/core/system_health/system_health.py

```diff
@@ -85,6 +85,22 @@ def get_node_status(self, node_id, **kwargs):
         """
         get node status method. This method is for returning a status of a node.
         """
+        try:
+            # Prepare key and read the health value.
+            key = self._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=node_id)
+            node_map_val = self.healthmanager.get_key(key)
+            if node_map_val is not None:
+                node_map_dict = eval(node_map_val)
+                key = self._prepare_key(component='node', cluster_id=node_map_dict['cluster_id'],
```

Comment:
create a const.py in ha/core/system_health and store all the string literals in it.

Reply:
const.py file created.